pythonforandroid/patching.py

```diff
@@ -69,3 +70,15 @@ def is_ndk(ndk):
     def is_x(recipe, **kwargs):
         return recipe.ctx.ndk == ndk
     return is_x
+
+def is_version_gt(version):
+    def is_x(recipe, **kwargs):
+        return LooseVersion(recipe.version) > version
+
+def is_version_gt(version):
+    def is_x(recipe, **kwargs):
+        return LooseVersion(recipe.version) < version
```

Comment:
The `is_version_gt()` function is defined twice, I guess one of the two should be `is_version_lt()`